raiden/tests/integration/test_events.py

```diff
@@ -11,6 +12,33 @@
 )
 from raiden.tests.utils.blockchain import wait_until_block
 from raiden.tests.utils.network import CHAIN
+from raiden.blockchain.abi import (
+    CHANNELNEWBALANCE_EVENTID,
+    CHANNELCLOSED_EVENTID,
+    CHANNELSETTLED_EVENTID,
+)
+
+
+def event_dicts_are_equal(dict1, dict2):
+    for k, v in dict1.iteritems():
+        if k not in dict2:
+            import pdb
+            pdb.set_trace()
```

Comment:
not sure if leaving the pdb trace call in here is the right thing to do. just raising should be enough -- developers can use `--pdb` flag in test sessions.